2020-03-18 09:50:41,551 [Thread-2: 12072]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/logout HTTP/1.1" 200 48
2020-03-18 09:50:43,158 [Thread-2: 12072]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/auth/authorize HTTP/1.1" 200 73
2020-03-18 10:36:26,282 [Thread-2: 14536]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\utils\log.py:log_response:228 WARNING - Not Found: /
2020-03-18 10:36:26,282 [Thread-2: 14536]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 WARNING - "GET / HTTP/1.1" 404 2031
2020-03-18 10:36:26,485 [Thread-2: 14536]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\utils\log.py:log_response:228 WARNING - Not Found: /favicon.ico
2020-03-18 10:36:26,486 [Thread-2: 14536]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 WARNING - "GET /favicon.ico HTTP/1.1" 404 2082
2020-03-18 10:37:04,240 [Thread-4: 18228]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/auth/authorize HTTP/1.1" 200 73
2020-03-18 10:39:15,939 [Thread-4: 18228]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/logout HTTP/1.1" 200 48
2020-03-18 10:39:16,967 [Thread-4: 18228]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/auth/authorize HTTP/1.1" 200 73
2020-03-19 14:45:50,003 [Thread-2: 14756]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\utils\log.py:log_response:228 ERROR - Internal Server Error: /api/v1.0/service/menu
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\handlers\base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "F:\微信小程序\Friday\school_engineer\backend\apis\views\menu.py", line 26, in get_menu
    global_app_data = init_app_data()
  File "F:\微信小程序\Friday\school_engineer\backend\apis\views\menu.py", line 21, in init_app_data
    with open(data_file, 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'F:\\微信小程序\\Friday\\school_engineer\\backend\\app.yaml'
2020-03-19 14:45:50,005 [Thread-2: 14756]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 ERROR - "GET /api/v1.0/service/menu HTTP/1.1" 500 71287
2020-03-19 14:46:44,498 [Thread-2: 19364]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 222
2020-03-19 14:48:38,152 [Thread-2: 5344]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:53:49,847 [Thread-3: 11164]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:55:29,996 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:57:15,980 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:57:29,380 [Thread-5: 14432]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 14:57:29,381 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 14:57:30,295 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 60
2020-03-19 14:57:30,313 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 60
2020-03-19 14:57:39,685 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:58:11,009 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 14:58:34,205 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:03:31,096 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:03:39,618 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/auth/authorize HTTP/1.1" 200 73
2020-03-19 15:07:26,468 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:09:07,170 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:10:19,927 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:11:42,857 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:13:04,707 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:13:49,577 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:15:24,301 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:16:12,575 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:16:46,255 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:17:07,081 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:17:10,115 [Thread-4: 2984]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:23:17,880 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:23:31,447 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:24:07,043 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:24:21,308 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 254
2020-03-19 15:24:36,313 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:24:53,087 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:25:00,349 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:25:12,801 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:25:20,603 [Thread-2: 5464]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:29:33,907 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:30:06,859 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:30:12,596 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:30:12,615 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:30:19,701 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:30:19,729 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:30:33,236 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:30:45,966 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:30:52,202 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:30:52,218 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:30:53,219 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:30:53,273 [Thread-2: 1944]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:32:35,506 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:35:01,133 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:35:50,127 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:36:05,363 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:40:33,061 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:42:05,505 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:42:22,859 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:43:13,319 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:47:36,040 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:47:45,466 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/menu HTTP/1.1" 200 236
2020-03-19 15:48:26,022 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:48:26,094 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:48:27,168 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:48:27,234 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 64
2020-03-19 15:48:36,063 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:48:47,136 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/service/sch_engineer HTTP/1.1" 200 40
2020-03-19 15:48:48,168 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 226
2020-03-19 15:48:55,695 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "POST /api/v1.0/auth/authorize HTTP/1.1" 200 73
2020-03-19 15:48:56,733 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/auth/status HTTP/1.1" 200 70
2020-03-19 15:48:57,585 [Thread-2: 7620]C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\django\core\servers\basehttp.py:log_message:136 INFO - "GET /api/v1.0/service/sch_engineer?type=backstage HTTP/1.1" 200 222
2020-03-19 08:17:26,106 [uWSGIWorker1Core0: 140563316180800]/home/ubuntu/.local/lib/python3.6/site-packages/django/core/handlers/exception.py:response_for_exception:74 ERROR - Invalid HTTP_HOST header: 'uwsgi'. You may need to add 'uwsgi' to ALLOWED_HOSTS.
